test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompaction_1_IT.java

```diff
@@ -500,6 +504,68 @@ public void testManytablets() throws Exception {
       compact(client, table1, 3, GROUP4, true);
 
       verify(client, table1, 3);
+
+      List<TabletId> tabletIds;
+      // start a compaction on each tablet
+      try (var tablets = client.tableOperations().getTabletInformation(table1, new Range())) {
```

Comment:
I wrote this test initially to check performance, but realized its a good test of many compaction on the same table w/ different config also.  

I did not initially see a performance difference w/ this test.  Looking into it the reason was the compaction were finishing too quickly, so only 8 or 9 compaction fate operations were active at a time.  For the performance bug to be observed needed many more active fate operations.  Locally I modified the test to kill the compactor processes and start 1K compaction fate operations.  With those changes I could see orders of magnitude diffs for fate lock acquisition time.